sample_consensus/include/pcl/sample_consensus/impl/sac_model_circle.hpp

```diff
@@ -171,23 +171,22 @@ pcl::SampleConsensusModelCircle2D<PointT>::selectWithinDistance (
   inliers.reserve (indices_->size ());
   error_sqr_dists_.reserve (indices_->size ());
 
+  const float sqr_inner_circle = (model_coefficients[2] <= threshold ? 0.0f : (model_coefficients[2] - threshold) * (model_coefficients[2] - threshold));
```

Comment:
Better name perhaps? Radius seems more apt than circle

Also some comments on checking if a point is inside 2 circles, with radius `R + r` and `R - r`, with `R` as `model_coeff[2]` and `r` as `threshold`. These are present in some but not all places

Reply:
I renamed the variables and added the comments. Please tell me if I missed any comments.